.github/BOTMETA.yml

```
@@ -1418,7 +1423,12 @@ files:
     <<: *docker
     support: community
   test/units/module_utils/facts/network/test_generic_bsd.py: *bsd
-  test/units/module_utils/hwc: *huawei
+  test/units/module_utils/hwc:
```

```suggestion
  test/units/module_utils/hwc: *huawei
```
. . . you can refer to it with an alias here and get rid of the duplicated lines.